1.1/terms_and_defns.ttl

```diff
@@ -0,0 +1,159 @@
+@prefix cs: <http://www.opengis.net/def/geosparql/td> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix td: <http://www.opengis.net/def/geosparql/td/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+cs:
+    a owl:Ontology , skos:ConceptScheme ;
+    dcterms:created "2021-06-22"^^xsd:date ;
+	dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
+    dcterms:modified "2021-06-22"^^xsd:date ;
+    dcterms:provenance "Each term (Concept) listed here indicates its own provenance since they are derived from other vocabularies and non-vocabulary sources."@en ;
+	dcterms:publisher [
+		a sdo:Organization ;
+		sdo:name "Open Geospatial Consortium" ;
+		sdo:url "https://www.ogc.org"^^xsd:anyURI ;
+	] ;
+    owl:versionInfo "OGC GeoSPARQL 1.1" ;
+    skos:definition "A vocabulary (taxonomy) of the Terms and Definitions listed by the GeoSPARQL 1.1 standard. These terms appear as a vocabulary resource (this file) and are also listed in Clause 4 of the GeoSPARQL 1.1 Specification"@en ;
+    skos:hasTopConcept
+        td:Subject ,
+        td:Predicate ,
+        td:Object ,
+        td:Resource ,
+        td:Class ,
+        td:Property ,
+        td:ObjectProperty ;
+        # td:DatatypeProperty ,
+        # td:AnnotationProperty 
+    skos:prefLabel "GeoSPARQL Terms & Definitions" ;
+.
+
+# Concept Template
+# td:XXX a skos:Concept ;
+#     dcterms:provenance ""@en ;
+#     rdfs:isDefinedBy <> ;
+#     skos:definition ""@en ;
+#     skos:inScheme cs: ;
+#     skos:prefLabel ""@en ;
+#     skos:topConceptOf cs: ;
+# .
+
+
+td:SemanticWeb a skos:Collection ;
```

Comment:
Is a skos:Collection the right thing to use here? It seems intended for array-like structures. Shouldn't we use broader/narrower relationship to form a hierarchy?

Reply:
We can use a Collection as an unordered bag (there is OrderedCollection too). I don't think we want hierarchy: of this whol vocab, there will be "domains" of terms like perhaps _Semantic Web_ & _Spatial_ etc. but the terms in them are unlikely to be hierarchically defined. Currently some of the terms here clearly have no hierarchical relationship, e.g.  `td:Subject`, `td:Class`, but are nevertheless still in the _Semantic Web_ domain.

I will add in hierarchy where appropriate (perhaps `td:Property skos:narrower td:ObjectProperty` etc. but hierarchies like that just won't cater for all the domain relations.

Reply:
Added hierarchy for some of the terms

Reply:
Am I right that you are saying that we want to capture different kinds of generalisation relationships? i.e.

1. terms that are defined within a knowledge domain (e.g. Feature is defined in the geoinformatics domain)
2. terms that are more specialised forms of other terms (e.g. Coordinate Reference Systems are a subset of Spatial Reference Systems)

I think it is possible to view both in the same way, as relationship between subsets and supersets. And that means only one mechanism for specifying generalisation relationships needs to be used. A compact example:
```
:GeosparqlTerms 
  a skos:ConceptScheme;
  skos:definition "Terms (and their definitions) used in the GeoSPARQL specification.";
  skos:hasTopConcept :SemWebTerms, :GeoinformaticsTerms
.
:SemWebTerms
  a skos:Concept;
  skos:definition "Terms (and their definitions) from the Semantic Web domain." ;
  skos:inScheme :GeoSparqlTerms
.
:Property
  a skos:Concept;
  skos:broader :SemWebTerms;
  skos:narrower :ObjectProperty, :DataProperty;
  skos:inScheme :GeoSparqlTerms
.
:SRS
  a skos:Concept;
  skos:broader :GeoinformaticsTerms;
  skos:narrower :CRS;
  skos:inScheme :GeoSparqlTerms
.
```
In all cases, `skos:broader` means "is a subset of" and skos:narrower means "is a superset of".

An alternative could be to model the set of terms from the Semantic Web and geoinformatics as concept schemes too, like below:

```
:GeoSparqlTerms 
  a skos:ConceptScheme;
  skos:definition "Terms (and their definitions) used in the GeoSPARQL specification.";
.
:SemWebTerms
  a skos:ConceptScheme;
  skos:definition "Terms (and their definitions) from the Semantic Web domain." ;
.
:SRS
  a skos:Concept;
  skos:narrower :CRS;
  skos:inScheme :GeoSparqlTerms, :SemWebTerms;
.
:Property
  a skos:Concept;
  skos:narrower :ObjectProperty, :DataProperty;
  skos:inScheme :GeoSparqlTerms, :SemWebTerms;
.
```

I think both ways of modelling are expressive enough without `skos:Collection`s. The second approach also has room for terms that are used in GeoSPARQL but do not really belong to either the semweb or geoinformatics knowledge domain.